plugins/who-can.yaml

```diff
@@ -3,23 +3,41 @@ kind: Plugin
 metadata:
   name: who-can
 spec:
-  version: "v0.1.0-alpha.1"
+  version: "v0.1.0-alpha.2"
   homepage: https://github.com/aquasecurity/kubectl-who-can
   shortDescription: |
-    Shows which users, groups and service accounts can perform a given action
-  description: |
-    Shows who has permissions to VERB [TYPE | TYPE/NAME | NONRESOURCEURL] 
+    Shows who has RBAC permissions to access Kubernetes resources
+  description: |+2
+    Shows which subject has RBAC permissions to VERB [TYPE | TYPE/NAME | NONRESOURCEURL] 
     
     VERB is a logical Kubernetes API verb like 'get', 'list', 'watch', 'delete', etc.
-    TYPE is a Kubernetes resource. Shortcuts, such as 'pod' or 'po' will be resolved. NAME is the name of a particular Kubernetes resource.
+    TYPE is a Kubernetes resource. Shortcuts, such as 'pod' or 'po' will be resolved.
+    NAME is the name of a particular Kubernetes resource.
     NONRESOURCEURL is a partial URL that starts with "/".
+
+    For example, if you want to find all the subjects who have permission to,
+    say, delete pods in a particular namespace, or to delete nodes in the
+    cluster (dangerous!) you could run the following commands:
+
+    $ kubectl who-can delete pods --namespace foo
+    $ kubectl who-can delete nodes
+
+    Without the who-can plugin you'd have to run a series of commands, and
```

Comment:
This last paragraph does not really add anything. After all, if it were present, nobody would write this plugin. But I'm not totally opposed to leaving it. I'm fine either way.